### PR TITLE
TopPhilicZp ttZp signal generation datacards

### PR DESCRIPTION
**Pull request by romeof**

### CODE REVIEW

**Comment by menglu21** [comment]
could you please remove the model, I have added them in the central place to be called by GEN script, and you can move README to the same directory with other cards. And finally please move those card to directory "2017/13TeV" to both master branch and "mg265UL"

**Comment by romeof** [comment]
Thanks for the comment. I have followed up on your suggestions.
Kindly let us know whether there is any other comment.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TopPhilicZp/ttzp/ttzp_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) 
+
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TopPhilicZp/ttzp/ttzp_proc_card.dat:44]: line 44 is identical with line 31, line 44 should be "add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) "

**Comment by menglu21** [comment]
Hi @romeof your current PR still includes many updates from other previous PRs, please update your local repo firstly

**Pull request closed without merging**